source/includes/fundamentals-sections.rst

```diff
@@ -19,4 +20,4 @@ Fundamentals section:
 
 .. - :doc:`Use the Driver's Data Formats </fundamentals/data-formats>`
 .. - :doc:`Record Events in the Driver </fundamentals/logging>`
-.. - :doc:`Use Driver Events in your Code </fundamentals/monitoring>`
\ No newline at end of file
+.. - :doc:`Use Driver Events in your Code </fundamentals/monitoring>`
```

Comment:
Suggestion: Not sure if you changed this line or not, but I think "your" should be capitalized.

https://www.mongodb.com/docs/meta/style-guide/style/titles-and-headings/#guidelines-for-headline-style-capitalization

Reply:
I agree and will update. It wasn't part of this PR, just a difficult-to-read GitHub diff representation of a whitespace change. If you click filter whitespace changes in the Files view, it won't show up.